test/Profiler/coverage_relative_path.swift

```diff
@@ -0,0 +1,18 @@
+// %s expands to an absolute path, so to test relative paths we need to create a
+// clean directory, put the source there, and cd into it.
+// RUN: rm -rf %t
+// RUN: mkdir -p %t/foo/bar/baz
+// RUN: cp %s %t/foo/bar/baz/coverage_relative_path.swift
+// RUN: cd %t/foo/bar
+
+// RUN: %target-swift-frontend -profile-generate -profile-coverage-mapping -emit-ir baz/coverage_relative_path.swift | %FileCheck -check-prefix=ABSOLUTE %s
```

Comment:
The master branch is tied to the old version of the coverage mapping format which doesn't support filename compression. However, master-next has the changes from https://reviews.llvm.org/D69471, so I suspect the test will break there. Could you add `-Xllvm -enable-name-compression=false` (or however it's spelled for swift)?

Reply:
I wondered about that given my other change, added here!